wikimedia/wikipedia-ios
# [YiR] Accessibility improvements

**Phabricator:** https://phabricator.wikimedia.org/T378129

### Notes
* Add some accessibility labels to images
* Add secondary style to WMFLarge button to use the larger tap area

### Test Steps
1. Got to YiR, turn on VoiceOver
2. Verify adjustments according to Robin's comment https://phabricator.wikimedia.org/T378129#10343948
